spec/features/user_can_add_tags_to_games_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+require 'helpers'
+
+feature 'user adds tags to game' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:tag) { FactoryGirl.create(:tag) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'user submits and adds new tag to game' do
```

Comment:
Since every test requires the user to be authenticated, you might want to move that to a `before :each` block:

```ruby
feature 'user adds tags to game' do
  let(:user) { FactoryGirl.create(:user) }

  before :each do
    sign_in_as(user)
  end
  
  scenario 'user submits and adds new tag to game' do
    # ...
  end
end
```